src/components/regionalization/RegionalizationModal/regionalization-modal.scss

```diff
@@ -0,0 +1,100 @@
+@import "src/styles/scaffold";
+
+[data-modal-overlay] {
```

Comment:
We are not scoping these styles in the component, so in this case, this will be duplicated. I believe we can remove it and keep the different background style.

![image](https://user-images.githubusercontent.com/3356699/160967339-bc494dae-e788-469a-adbc-65941f6b8612.png)